### PR TITLE
[RM RA1] Add content for Containerisation Support / RM citations

### LINKED ISSUE #3183: [RA1Ch04] Containerized applications on OpenStack

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #3183 
Further to the discussion in the RA1 meeting on 2022-11-21, added:
1. RA1 Chapter 4, subsection on "Containerisation Support" - refers to RA2 Chapter 4.
2. Added reference to RA2 and Kubernetes in the refs.bib file

Also, converted RM references to citations

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter04.rst [hunk 1]
```diff
@@ -1399,6 +1399,21 @@ the figure below.
 
    Containerised OpenStack Services Topology
 
+Containerisation Support
+^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (karinesevilla) [inline doc/ref_arch/openstack/chapters/chapter04.rst:1403]: @pgoyal01 This paragraph is under the section "Containerised OpenStack Services". As it deals with workloads and not OpenStack services, the first title "Containerised OpenStack Services" can be changed by "Containerisation" or something more general  and 2 sub-sections can be created one for "Containerised OpenStack Services and the other one for "Containerised workloads" .
I agree with the content

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter04.rst]: @karinesevilla Addressed changes by creating a high-level section and two sub-sections

**Review by collivier — APPROVED** [review]

**Comment by pgoyal01** [comment]
@walterkozlowski @karinesevilla @collivier I made a mistake and from my repo I added the RM citation changes to the already open PR.

So should only be merged with enough RM and RA1 approvals.

**Review by walterkozlowski — APPROVED** [review]

File: doc/ref_model/chapters/chapter04.rst [hunk 2]
```diff
@@ -269,62 +269,62 @@ or events should be collected and monitored by monitoring tools.
 |         | Infrastructure      |           |                                                                          |
 |         | Capability          |           |                                                                          |
 +=========+=====================+===========+==========================================================================+
-|i.pm.001 | Host CPU usage      |nanoseconds| Per Compute node. It maps to ETSI GS NFV-TST 008 V3.5.1 [5] clause 6,    |
+|i.pm.001 | Host CPU usage      |nanoseconds| Per Compute node. It maps to ETSI GS NFV-TST 008 V3.5.1 :cite:p:`etsigsnfvtst008` clause 6,    |
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter04.rst:272]: I think this breaks the grid tables. You need to fit the text to the grid. 
This comment is valid for all similar cases.

Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter04.rst:272]: I double checked and yes, this breaks the tables and RM does not compile. 
As you might also recognized the automatic builds are not working anymore, so I would like to ask the sub-project leads to ensure that the proposed changes are not breaking the build before they get merged. Thanks.

**Pull request merged**